### ISSUE TITLE
Inconsistent behaviour while deleting buffer

### ISSUE THREAD

**Issue by Parth576** [Kethku/neovide#569]
I delete a buffer using the ```:bd``` command. Sometimes, the buffer closes as expected, but sometimes Neovide itself shuts down/crashes.

**Comment by Kethku** [comment]
Thats not good. What version of neovim are you using? Are you sure it doesn't happen with nvim in the terminal? What platform are you on?

**Comment by Parth576** [comment]
`NVIM v0.4.4
Build type: Release
LuaJIT 2.0.`

Sorry, I had forgotten to check with nvim in the terminal, been using neovide since I had installed it :sweat_smile: . But yes, I was able to reproduce this error in the terminal. Looks like something to do with NERDTree. This is what I got from the log.

![10-47|20April2021](https://user-images.githubusercontent.com/27119104/115341255-e35bb700-a1c5-11eb-80cb-2b592ca76448.png)
